server/modlog.ts

```diff
@@ -352,14 +351,22 @@ export class Modlog {
 	}
 }
 
-export const PM = new QueryProcessManager<ModlogQuery, string[] | undefined>(module, async data => {
+// if I don't do this TypeScript thinks that (ModlogResult | undefined)[] is a function
+// and complains about an "nexpected newline between function name and paren"
+// even though it's a type not a function...
```

Comment:
I've had this problem too! I'm in the process of upgrading eslint which should hopefully fix this.